brooklyn-server/server-cli/src/main/resources/brooklyn/default.catalog.bom

```diff
@@ -240,6 +242,8 @@ brooklyn.catalog:
               # and a lot of sensors defined
               type:           2-bash-web-server-template
               name:           My Bash Web Server VM with Sensors
+              # and clear the location defined there so it is taken from this template
+              locations:      []
```

Comment:
on first sight I'm bit torn on this, its unambiguous but the 0.7.0-incubating behaviour felt very slick..